test/regservice_selection_dialog.rb

```diff
@@ -0,0 +1,103 @@
+require_relative "spec_helper"
```

Comment:
@imobachgs Only now I have noticed that this file does not end in _spec or _test (there are both kinds in this repo, ugh!) so it is not included in `rake test:unit`.